frontend/src/components/Forum/Post.jsx

```
@@ -64,64 +67,107 @@ const Post = ({ post, onDelete }) => {
+        toggleComments={toggleComments}
+        commentCount={updatedPost.comments?.length || 0}
       />
 
       {/* Edit & Delete Buttons (Only for Author) */}
       {isAuthor && (
-        <div className="flex gap-4 mt-4">
-          <button onClick={openEditModal} className="text-blue-600 hover:underline font-medium">Edit</button>
-          <button onClick={handleDelete} className="text-red-600 hover:underline font-medium">Delete</button>
+        <div className="flex gap-4">
+          <button onClick={openEditModal} className="text-blue-600 hover:underline font-medium">
+            Edit
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)